src/components/settings/SettingsLMSTab/LMSConfigs/DegreedConfig.jsx

```diff
@@ -1,14 +1,16 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 import { Button, Form, useToggle } from '@edx/paragon';
+import isEmpty from 'lodash/isEmpty';
+
 import { buttonBool, handleErrors } from '../LMSConfigPage';
 import LmsApiService from '../../../../data/services/LmsApiService';
 import { snakeCaseDict, urlValidation } from '../../../../utils';
 import ConfigError from '../ConfigError';
 import ConfigModal from '../ConfigModal';
 import { INVALID_LINK, INVALID_NAME, SUCCESS_LABEL } from '../../data/constants';
 
-const DegreedConfig = ({ id, onClick }) => {
+const DegreedConfig = ({ enterpriseCustomerUuid, onClick, existingData }) => {
```

Comment:
nit: optional: I think naming these onSuccess instead of onClick better represents what it does in this component ( same applies to all )

Reply:
🤔 agreed